confik/tests/third_party.rs

```diff
@@ -126,3 +126,76 @@ mod bigdecimal {
         }
     }
 }
+
+#[cfg(feature = "js_option")]
+mod js_option {
+    use confik::{Configuration, TomlSource};
+    use js_option::JsOption;
+
+    #[derive(Configuration, Debug)]
+    struct Config {
+        opt: JsOption<usize>,
+    }
+
+    #[test]
+    fn undefined() {
+        let config = Config::builder()
+            .try_build()
+            .expect("Should be valid without config");
+        assert_eq!(config.opt, JsOption::Undefined);
+    }
```

Comment:
nice, can we also assert the behavior for TOML to show that it produces Undefined and Some ?

Reply:
meh, nah i take it back, it's not really useful to do that